janusgraph-test/src/test/java/org/janusgraph/graphdb/tinkerpop/JanusGraphIoRegistryTest.java

```diff
@@ -60,7 +60,7 @@ public void testTinkerPopPredicatesAsGryo() throws SerializationException {
         assertEquals(15,
                 Stream.of(P.class.getDeclaredMethods()).filter(m -> Modifier.isStatic(m.getModifiers())).filter(p -> {
                     log.debug("Predicate: {}", p);
-                    return true;
+                    return !p.isSynthetic();
```

Comment:
@robertdale Since you added this test (and comment) does the update here seem okay?